MODULE.bazel

```diff
@@ -32,6 +32,11 @@ bazel_dep(name = "googletest", version = "1.16.0")
 bazel_dep(name = "google_benchmark", version = "1.9.1")
 bazel_dep(name = "abseil-cpp", version = "20250127.0", repo_name = "com_google_absl")
 
+# Yosys dependencies that are already on the bazel central registry
+# abc can't be used in place because it changes the names of the targets from abc-lib -> abc and abc -> abc_bin
+# bazel_dep(name = "abc", version = "0.0.0-20250408-yosyshq.bcr.1", repo_name = "edu_berkeley_abc")
```

Comment:
Oh! I might be able to handle this, in doing the fully-homomorphic-encryption stuff I am again familiarizing myself with the dep issues.